app/helpers/application_form_helper.rb

```diff
@@ -0,0 +1,35 @@
+module ApplicationFormHelper
```

Comment:
I generally prefer going with view objects rather than using helpers like this. Happy to discuss and not do that as I realise using helpers is pretty established. It just feels cleaner to me to have things encapsulated in an object rather than a module that gets included all over the place. No naming issues, you only have to pass the object in once rather than to every method 🤷

Reply:
Yeah, I do generally agree with you. I've been trying to stick with standard Rails patterns until they become unmaintainable/hard to scale before switching to more bespoke patterns. I decided to go with helpers here as they're supported natively in view components and it didn't quite feel big enough to cause problems. I also think it would be better to not have them automatically included in each controller and include them manually, which we could configure Rails to do.

I'm happy to switch to view objects, do we have a consensus over where the files should be put in the repo? I also wonder if the existing helpers should be refactored to use view objects at the same time so I wonder about doing that in a separate PR - what do you think?

Reply:
Ok cool. Let's merge this and refactor them all in another PR. I'll card that up.